msw/msw-app.sh

```diff
@@ -29,7 +29,7 @@ BUILD=..
 #----------------------------------------------------------
 help() {
 cat <<EOF
-Usage: msw-app.sh [OPTIONS] [VERSION]
+Usage: $0 [OPTIONS] [VERSION]
```

Comment:
I'm not a fan of this as it will print the whole path when called from another directory and screw up the formatting.